demo-notebooks/guided-demos/2_basic_jobs.ipynb

```diff
@@ -116,7 +116,13 @@
    "id": "83d77b74",
    "metadata": {},
    "source": [
-    "First, let's begin by submitting to Ray, training a basic NN on the MNIST dataset:"
+    "First, let's begin by submitting to Ray, training a basic NN on the MNIST dataset:\n",
+    "\n",
+    "NOTE: To test this demo in an air-gapped/ disconnected environment alter the training script to use a local dataset.\n",
```

Comment:
Should we mention that, in this example, the Python dependencies are installed in the custom image, and that's mandatory for it to work in disconnected environments?

Reply:
Should we upload the image with dependency mirror to Quay so we can reference it here?

Reply:
Would be even better yes.

Reply:
thinking, will users always follow the docs for disconnected testing, or will they use directly the notebook, skipping docs?

Reply:
you're probably right, it wouldn't hurt to have them in both places